src/components/users/ForgotPwd.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from 'react';
+import { func, object } from 'prop-types';
+import Loader from 'react-loader-spinner';
+import { useDispatch, useSelector } from 'react-redux';
+
+import '../../style/App.scss';
```

Comment:
As a good practice, try to avoid relative imports that are not in the same folder

Reply:
Ok I'll change this in another PR